src/components/avatar/index.tsx

```diff
@@ -136,7 +137,7 @@ export type AvatarProps = {
    * Used as a testing identifier
    */
   testID?: string;
-};
+} & AccessibilityProps;
```

Comment:
1. Do you need them all? Perhaps you can [Pick the way we do in Dialog](https://github.com/wix/react-native-ui-lib/blob/master/src/components/dialog/index.tsx#L22-L24)?
2. Either way, it should be at the start (`export type AvatarProps = AccessibilityProps & {`).